aws_lambda_powertools/utilities/idempotency/persistence/dynamodb.py

```diff
@@ -114,6 +115,14 @@ def __init__(
         self.data_attr = data_attr
         self.validation_key_attr = validation_key_attr
 
+        # Use DynamoDB's ReturnValuesOnConditionCheckFailure to optimize put and get operations and optimize costs.
+        # This feature is supported in boto3 versions 1.26.164 and later.
+        self.support_return_on_check_failure = (
```

Comment:
One last thing, when reading this variable my mind thinks it will be a boolean (supports or not supports) but that's not the case. What about:

```suggestion
        self.return_value_on_condition = (
```

? what do you think?

Reply:
Yes! It makes sense! I changed.